2020-January.html

```diff
@@ -0,0 +1,247 @@
+<!doctype html>
+<html lang="en">
+
+    <head>
+        <meta charset="utf-8">
+
+        <title>AmsterdamPHP</title>
+
+        <meta name="author" content="AmsterdamPHP Enablers">
+
+        <meta name="apple-mobile-web-app-capable" content="yes" />
+        <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+        <link rel="stylesheet" href="css/reveal.min.css">
+        <link rel="stylesheet" href="css/theme/amsterdamphp.css" id="theme">
+
+        <!-- For syntax highlighting -->
+        <link rel="stylesheet" href="lib/css/zenburn.css">
+
+        <!-- Lobster font -->
+        <link href='https://fonts.googleapis.com/css?family=Lobster' rel='stylesheet' type='text/css'>
+
+        <!-- Font awesome for icons -->
+        <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
+
+        <!-- If the query includes 'print-pdf', include the PDF print sheet -->
+        <script>
+            if( window.location.search.match( /print-pdf/gi ) ) {
+                var link = document.createElement( 'link' );
+                link.rel = 'stylesheet';
+                link.type = 'text/css';
+                link.href = 'css/print/pdf.css';
+                document.getElementsByTagName( 'head' )[0].appendChild( link );
+            }
+        </script>
+
+        <!--[if lt IE 9]>
+        <script src="lib/js/html5shiv.js"></script>
+        <![endif]-->
+
+        <script>
+            slideData = {
+                "text": {
+                    "date": "January 2020",
+                    "speaker-name": "Thijs de Maa",
+                    "next-meetup-date": "20th of February"
+                },
+                "images": {
+                    "hoster-small": {
+                        "src": "img/hoster-logos/castor.svg",
+                        "alt": "Hosted by Castor"
+                    },
+                    "hoster-large": {
+                        "src": "img/hoster-logos/castor.svg",
+                        "alt": "Hosted by Castor"
+                    },
+                    "hoster-next": {
+                        "src": "img/hoster-logos/castor.svg",
```

Comment:
Next hoster isn't Castor, right?

Reply:
Couldn't find the next one on the sheet. But didn't have time yet to ping slack :P

Reply:
Dropped the next hoster for now